src/bindings/python/tests/test_graph/test_custom_op.py

```diff
@@ -197,3 +199,89 @@ def test_custom_op():
     input_tensor = Tensor(input_data)
     results = request.infer({"data": input_tensor})
     assert np.allclose(results[list(results)[0]], expected_output, 1e-4, 1e-4)
+
+
+class CustomSimpleOp(Op):
```

Comment:
Could you add a test that checks type_info for this op to show how it is auto-generated?